test/logging/TestLogQueue.cpp

```diff
@@ -85,8 +85,8 @@ TEST_F(LogQueueTest, notifyAllOnShutdown)
 
     logQueue->shutdown();
 
-    cv.wait_for(lock1, chrono::seconds(1));
-    cv2.wait_for(lock2, chrono::seconds(1));
+    cv.wait_for(lock1, chrono::milliseconds(400));
```

Comment:
[Minor] I guess you can make it `2 * EMPTY_WAIT_TIME_MILLISECONDS` to be a little more explicit about the connection between this wait and the code that it relies on, but I will leave it to you to decide.

Reply:
since EMPTY_WAIT_TIME_MILLISECONDS is a private member, I'll leave it as is and add a comment for explanation